service/matching/physical_task_queue_manager.go

```diff
@@ -245,6 +350,7 @@ func (c *physicalTaskQueueManagerImpl) AddTask(
 	ctx context.Context,
 	params addTaskParams,
 ) (bool, error) {
+	c.TasksAddedInIntervals.trackTasks()
```

Comment:
AddTask is removed from physicalTaskQueueManager in main branch. Please move this line to trySyncMatch. and put it in the first if block so it is not double-counted for forwarded tasks.

Also, we need to count the Query and Nexus tasks that are added, by calling `c.TasksAddedInIntervals.trackTasks` in DispatchQueryTask and DispatchNexusTask methods. Note that those ones should also only track when request.forwardedFrom is empty. No need to do anything extra for counting dispatch of nexus and query tasks as they also use the Poll method.